lib/config/configuration.go

```diff
@@ -1963,8 +1969,10 @@ func Configure(clf *CommandLineFlags, cfg *servicecfg.Config, legacyAppFlags boo
 		cfg.Databases.Databases = append(cfg.Databases.Databases, db)
 	}
 
-	if err = ApplyFileConfig(fileConf, cfg); err != nil {
-		return trace.Wrap(err)
+	if !cfg.OpenSSH.Enabled {
```

Comment:
One thing I'm not really a fan of is that we're updating "Configure" method to have a special behavior if OpenSSH "mode" is enabled, I think Edoardo raised a similar concern.

Instead of changing this function to check `cfg.OpenSSH.Enabled` in several places and updating the config "from within" using "applyOpenSSHConfig" below, can we keep this function as-is, and just have a dedicated "ConfigureOpenSSH" method which would construct config specifically for OpenSSH mode (basicaly, do what "applyOpenSSHConfig" does). Cause you don't need probably 90% of what's in this function for OpenSSH (like, databases, apps and other services).